modules/ossm-observability-access.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+//- ossm-observability.adoc
+
+[id="ossm-observability-access-console_{context}"]
+= Accessing the Kiali console
+
+To access the console, in the menu bar, click the *Application launcher* > *Kiali*.
+
+Select the project for your service in the *Namespace* field. Log in to the Kiali console with the same user name and password as you use to access the {product-title} console. If you have installed the Bookinfo example, select `bookinfo`.
+
+.Procedure from the command line
+
+. Run this command from the CLI to obtain the route and Kiali URL:
++
+----
+$ oc get routes
+----
++
+In the output on the `kiali` line, use the URL in the HOST/PORT column to open the Kiali console. Select the project for your service in the *Namespace* field. Log in to the Kiali console with the same user name and password as you use to access the {product-title} console.
```

Comment:
Same comment about the order of the login step and selecting a namespace.

Reply:
made a change